pkg/providers/launchtemplate/suite_test.go

```diff
@@ -1799,6 +1829,12 @@ var _ = Describe("LaunchTemplate Provider", func() {
 				)
 			})
 			It("should set LocalDiskStrategy to Raid0 when specified by the InstanceStorePolicy", func() {
+				nodePool.Spec.Template.Spec.Requirements = append(nodePool.Spec.Template.Spec.Requirements, karpv1.NodeSelectorRequirementWithMinValues{
```

Comment:
Why do we need to add this requirement? Is there additional filtering being done where this wouldn't be set for non-instance store instances that I'm missing?

Reply:
We don't have to. I'm fine to drop it. This was an artifact of me adding userData that would selectively configure RAID0. Since we are dropping that, I don't think there is a need for this anymore.